proposals/vector_buffer.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "vector_buffer",
+    "summary": "Compute a buffer around geometries",
+    "description": "Calculates a round buffer around each geometry given. Buffers can be applied to points, lines and polygons, but the results are always polygons. Multi-part types (e.g. `MultiPoint`) are also allowed.",
```

Comment:
Having a non-GIS background, I found the term "buffer" a bit strange at first. I think it would be good if the description contained other keywords too like "morphological operation", "dilation"(="outward buffer"), "erosion" ("inward buffer")

Reply:
Sure, that would be appreciated, I think. Would you mind providing a proposal?

Reply:
Okay, I've added something. What do you think?